features/claim_based_routing.md

```diff
@@ -0,0 +1,235 @@
+[//]: # (The syntax preceeding this line is a comment marker used to help guide the author in populating this document)
+[//]: # (to github. Unlike HTML comments commonly used throughout istio.io documentation, this comment will not be rendered)
+[//]: # (by github. Comments must be separated by carriage return preceding and concluding the text and be a single line.)
+
+[//]: # (This is a living document representing the maturity of a feature. Completion of this template enables Istio work groups)
+[//]: # (to collect information on potential new functionality. This template should be completed before users are exposed to)
+[//]: # (any new experimental feature. Please complete this template during development.)
+
+[//]: # (The feature implementation section must be completed before submission of the document.)
+
+# Feature:
+
+[//]: # (All information in this section is mandatory.)
+
+**Feature name:**
+
+JWT Claim Based Routing
```

Comment:
I can't do suggestions (maybe as this is merged), and I think the TOC might want to provide some guidance on how to accurately fill these form out. So of the next time we want to update this document, based on some early docs that Brian did, I think the lines in the header can be on a single line, so something like:

**Feature name:**  JWT Claim Based Routing

for this and following header sections.